mobile/test/features/student_management/presentation/bloc/student_bloc_test.dart

```diff
@@ -99,30 +99,35 @@ void main() {
       bloc.add(CreateNewStudentEvent(
         firstName: tFirstName,
         lastName: tLastName,
+        classroom: tClassroom,
       ));
     });
   });
 
   group('updateStudent', () {
-    test('''should emit [UpdatingStudent, StudentUpdated] when update 
```

Comment:
I think this test is not entirely correct. We should test with an `updatedStudent` to check if the correct changes are being updated